pages/2020/index.rst

```diff
@@ -0,0 +1,111 @@
+.. title: Python in Astronomy 2020
+
+20 - 24 April 2020
+--------------------
+
+Trinity College, Dublin
+-----------------------
+
+.. image:: /images/pyastro_logo_150px.png
+   :align: center
+   :width: 150px
+
+The sixth Python in Astronomy Conference will be held at Trinity College,
+Dublin on **20 - 24 April 2020**.
+
+This conference series aims to bring together a wide variety of developers,
+educators and other contributors to Python packages in the context of all
+forms of astronomy. The conference will include unconference sessions,
+presentations, sprints and tutorials. As well as expanding the community around
+astronomical uses of Python, the conference aims to improve collaboration and
+interoperability between Python packages, and to share knowledge on Python
+packages and techniques.
+
+All participants at the conference will be expected to follow the
+`code of conduct </code-of-conduct>`_.
+
+More details will be made available on this page closer to the week of the
+conference.
+
+Applying
+########
+
+A link to the application page will be posted here in the next few weeks.
+
+
+..
+   Proceedings
+   ###########
+
+Program
+#######
+
+Once finalized, the conference schedule will be posted here.
+
+..
+   The conference schedule and related info `can be found here </2020/schedule>`_.
+
+Chat and Social Media
+#####################
+
+We will be using both Matrix and Slack for online chat throughout the week,
+with a bridge to sync messages from one to the other, so people can use
+whichever platform they prefer.
+You can `go here to get an invitation for the Slack team <https://join.slack.com/t/pyastro/shared_invite/enQtNjk4MjMxNjE3Mzk1LWZkMGI1N2QxOTBlODYzNjk4OGIzMGI0MGU5YjQ2M2Y3ZTM1MTI4ZGU5MmM1ZGIyYzgxYjFmOThiZDg0ZGI5YjY>`_,
```

Comment:
does this invite still work?

Reply:
Is there any reason why it shouldn't? The team hasn't changed.

Reply:
I thought they expire.

Reply:
Oh possibly. I'll check.